src/plugins/home/config.ts

```diff
@@ -33,6 +33,7 @@ import { schema, TypeOf } from '@osd/config-schema';
 export const configSchema = schema.object({
   disableWelcomeScreen: schema.boolean({ defaultValue: false }),
   disableNewThemeModal: schema.boolean({ defaultValue: false }),
+  newHomepage: schema.boolean({ defaultValue: false }),
```

Comment:
nit: can maybe mimic the other configurations like `disableNewHomePage`

I wonder since since the default prompts are verify specific, i wonder if we should just make default `true`.

Reply:
+1, lets follow current scheme of the config naming here.